docs/tutorials/random.rst

```diff
@@ -15,69 +14,69 @@ name across the display::
 
-Every time the device is reset it displays a number between 1 and 6. You're
-starting to get familiar with *nesting*, so it's important to note that
-``random.randint`` returns a whole number between the two arguments, inclusive
-(a whole number is also called an integer - hence the name of the method).
-Notice that because ``display.show`` expects a character then we use the
-``str`` function to turn the numeric value into a character (we turn, for
-example, ``6`` into ``"6"``).
-
-If you know you'll always want a number between ``0`` and ``N`` then use the
-``random.randrange`` method. If you give it a single argument it'll return
-random integers up to, but not including, the value of the argument ``N``
-(this is different to the behaviour of ``random.randint``).
-
-Sometimes you need numbers with a decimal point in them. These are called
-*floating point* numbers and it's possible to generate such a number with the
-``random.random`` method. This only returns values between ``0.0`` and ``1.0``
-inclusive. If you need larger random floating point numbers add the results
-of ``random.randrange`` and ``random.random`` like this::
+Po każdym restarcie urządzenia wyświetla ono numer z zakresu 1-6. Zaczynasz
+zapoznawać się z *zagnieżdżaniem*. Warto więc tylko zauważyć, że ``random.randint``
+zwraca liczbę całkowitą z przedziału domkniętego pomiędzy dwoma argumentami
+(liczba całkowita nazywana jest integer, stąd nazwa metody). Zauważ, że skoro
+``display.show`` oczekuje znaku alfabetycznego, używamy funkcji ``str`` do zamiany
+numeru na znak (na przykład: ``6`` w ``"6"``).
+
+W wypadku, gdy zawsze potrzebujesz liczny z przedziału pomiędzy ``0`` a ``N``,
+używaj metody ``random.randrange``. Gdy podasz jej pojedynczy argument, zwróci
+losową liczbę całkowitą z przedziału zamkniętego od ``0`` do ``N`` (w
+przeciwieństwie do funkcji ``random.randint``).
+
+Czasami potrzebujesz liczby z miejscem po przecinku. Nazywamy je liczbami
+*zmiennoprzecinkowymi* i można je uzyskać używając metody ``random.radom``.
+Zwraca ona wartość z przedziału domkniętego od ``0.0`` do ``1.0``. Jeżeli
+potrzebujesz dużej liczby zmiennoprzecinkowej dodaj ``random.randrange`` do
+``random.random``, jak poniżej::
 
     from microbit import *
     import random
 
     answer = random.randrange(100) + random.random()
     display.scroll(str(answer))
 
-Seeds of Chaos
+Pula Chaosu
 ++++++++++++++
 
-The random number generators used by computers are not truly random. They just
-give random like results given a starting *seed* value. The seed is often
-generated from random-ish values such as the current time and/or readings from
-sensors such as the thermometers built into chips.
+Losowe liczby generowane przez komputery nie są naprawdę losowe. Zwracają
+jedynie pseudolosowe rezultaty zaczynając od wartości początkowej. Wartość
+początkowa generowana jest na podstawie pseudolosowej wartości, takiej jak
+obecna godzina i/lub odczyty z sensorów sprzętowych.
```

Comment:
"Samo ziarno generowane jest często na podstawie trudnych do przewidzenia z góry danych, takich jak..." — wiem, że w oryginale jest inaczej, ale to wydaje mi się bardziej precyzyjne i jaśniejsze